lua/pckr/loader/event.lua

```diff
@@ -3,14 +3,24 @@
 --- @return fun(_: fun())
 return function(events, pattern)
   return function(loader)
+    local done = false
     vim.api.nvim_create_autocmd(events, {
       pattern = pattern,
       once = true,
       desc = 'pckr.nvim lazy load',
-      callback = function()
+      callback = function(ev)
+        if done then
```

Comment:
This has `once = true`, so this should never be executed twice.

Reply:
https://github.com/folke/lazy.nvim/blob/main/lua/lazy/core/handler/event.lua#L76
https://github.com/neovim/neovim/issues/25526

This is work-around for issue above.
After my testing, nvim 0.10 does indeed require this work-around.